packages/dashboard/src/components/run/details.tsx

```diff
@@ -27,25 +36,120 @@ export function RunDetails({ run }: RunDetailsProps) {
           onChange={() => setHidePassedSpecs(!isPassedHidden)}
         />
       </HFlow>
-      <ul>
-        {specs
-          .filter((spec) => !!spec)
-          .filter((spec) =>
-            isPassedHidden ? getSpecState(spec!) !== 'passed' : true
-          )
-          .map((spec) => (
-            <li
-              key={spec!.instanceId}
-              className={css`
-                 {
-                  padding: 12px 0;
+      <div className={css`
+        {
+          margin: 12px 0;
+        }
+      `}>
+        <DataTable
+          rows={
+            specs.filter((spec) => !!spec)
+              .filter((spec) =>
+                isPassedHidden ? getSpecState(spec!) !== 'passed' : true
+              )
+          }
+          loading={false}
+          columns={[
+            {
+              name: 'status',
+              header: 'Status',
+              sortable: false,
+              render: spec => <SpecState state={getSpecState(spec)} />,
+            },
+            {
+              name: 'link',
+              header: '',
+              sortable: false,
+              render: spec => <Link href={generatePath(`/instance/${spec?.instanceId}`)}>{spec?.spec}</Link>,
+            },
+            {
+              name: 'durration',
+              header: 'Durration',
+              sortable: false,
+              render: spec => {
+                if (spec?.results?.stats?.wallClockDuration) {
+                  return (
+                    <Tooltip text={`Started at ${spec.results.stats.wallClockStartedAt}`}>
+                      <Text>{shortEnglishHumanizerWithMsIfNeeded(spec.results.stats.wallClockDuration, miniutesAndSecondsOptions)}</Text>
+                    </Tooltip>
+                  )
+                } else if (spec.claimedAt) {
+                  return (
+                    <Tooltip text={`Started at ${spec.claimedAt}`}>
+                      <Text>
+                        <RenderOnInterval live refreshIntervalInSeconds={1} renderChild={()=>{
```

Comment:
This makes the durration on the page update every second until tests have completed.